spec/spec_helper.cr

```diff
@@ -8,7 +8,7 @@ require "./support/**"
 TEST_PATH    = "spec/support/sample"
 PUBLIC_PATH  = TEST_PATH + "/public"
 VIEWS_PATH   = TEST_PATH + "/views"
-TESTING_APP  = "./test_app"
+TESTING_APP  = "./tmp/test_app"
```

Comment:
Possibly but because of permissions on in CI's and things I think it probably makes more sense to use tmp within the project which is in the git ignore. Rails makes use of this pattern as well.